frontend/app/src/pages/role-management/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { Card } from "@/components/ui/card";
+import Content from "@/screens/layout/content";
+import { RoleManagementRoot } from "@/screens/role-management";
+
+function RoleManagement() {
+  return (
+    <Content className="p-2">
+      <Card className="p-0 overflow-hidden">
+        <Content.Title
+          title={"Role Management"}
+          description="Accounts, groups, roles and permissions management"
+          className="border-b-0"
+        />
+
+        <RoleManagementRoot />
```

Comment:
how about renaming it to <RoleManagementNavigation /> and keep Outlet out of it like:
```tsx
function RoleManagement() {
  return (
    <Content className="p-2">
      <Card className="p-0 overflow-hidden">
        <Content.Title
          title={"Role Management"}
          description="Accounts, groups, roles and permissions management"
          className="border-b-0"
        />

        <RoleManagementNavigation />
        <Outlet />
      </Card>
    </Content>
  );
}

```